### PR TITLE
Typo in web frameworks doc

### PR DESCRIPTION
**Pull request by omareg94**
Fixed "Model-View-Controller" link

> What was wrong/why is this fix needed? (quick summary only)

> MDN URL of main page changed

> Issue number (if there is an associated issue)

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/learn/server-side/first_steps/web_frameworks/index.html [hunk 1]
```diff
@@ -178,7 +178,7 @@ <h2 id="How_to_select_a_web_framework">How to select a web framework</h2>
...
-   <li><em>Whether or not the framework encourages good development practices</em>: For example, a framework that encourages a <a href="/en-US/docs/Web/Apps/Fundamentals/Modern_web_app_architecture/MVC_architecture">Model-View-Controller</a> architecture to separate code into logical functions will result in more maintainable code than one that has no expectations on developers. Similarly, framework design can have a large impact on how easy it is to test and re-use code.</li>
+   <li><em>Whether or not the framework encourages good development practices</em>: For example, a framework that encourages a <a href="https://developer.mozilla.org/en-US/docs/Glossary/MVC">Model-View-Controller</a> architecture to separate code into logical functions will result in more maintainable code than one that has no expectations on developers. Similarly, framework design can have a large impact on how easy it is to test and re-use code.</li>
```
Comment (Ryuno-Ki) [inline files/en-us/learn/server-side/first_steps/web_frameworks/index.html:181]:
```suggestion
   <li><em>Whether or not the framework encourages good development practices</em>: For example, a framework that encourages a <a href="/en-US/docs/Glossary/MVC">Model-View-Controller</a> architecture to separate code into logical functions will result in more maintainable code than one that has no expectations on developers. Similarly, framework design can have a large impact on how easy it is to test and re-use code.</li>
```

We don't put the domain into internal links :)

**Review by Ryuno-Ki — APPROVED** [review]
`/Apps` points to PWA, which is not the right context for this. The previous link is broken.

In other words: Thank you for catching this!

**Comment by Ryuno-Ki** [comment]
The pr-linter fails with: „Pull request title can't just be "Update index.html"”, but I must have missed our current conventions.
If you look at other PRs, you might figure out the format. Then, go into the upper right corner and hit the Edit button (next to „Open with”) of this PR to update the PR title.

**Review by hamishwillee — APPROVED** [review]
Thanks very much @omareg94 (and @Ryuno-Ki - I committed your patch).

As noted, MDN internal URLs start from `/en-US/...` (in the English version).

**Pull request merged**